src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
                 <div class="chi-button__content">
                   <i class="chi-icon icon-x"></i>
                 </div>
               </button>
             </div>
-            <div class="chi-drawer__content -px--2 -py--3">
-              <div class="chi-form__item -mb--2">
-                <label class="chi-label" for="example-ba11">Label</label>
-                <select class="chi-select" id="example-ba11">
-                  <option value="">Select</option>
-                  <option>Option 1</option>
-                  <option>Option 2</option>
-                  <option>Option 3</option>
-                </select>
-              </div>
-              <div class="chi-form__item -mb--2">
-                <label class="chi-label" for="example-ba12">Label</label>
-                <select class="chi-select" id="example-ba12">
-                  <option value="">Select</option>
-                  <option>Option 1</option>
-                  <option>Option 2</option>
-                  <option>Option 3</option>
-                </select>
-              </div>
-              <div class="chi-form__item -mb--2">
-                <label class="chi-label" for="example-ba13">Label</label>
-                <select class="chi-select" id="example-ba13">
-                  <option value="">Select</option>
-                  <option>Option 1</option>
-                  <option>Option 2</option>
-                  <option>Option 3</option>
-                </select>
-              </div>
-              <div class="chi-form__item -mb--2">
-                <label class="chi-label" for="example-ba14">Label</label>
-                <select class="chi-select" id="example-ba14">
-                  <option value="">Select</option>
-                  <option>Option 1</option>
-                  <option>Option 2</option>
-                  <option>Option 3</option>
-                </select>
-              </div>
-            </div>
-            <div class="-d--flex -justify-content--center -pb--2">
-              <button class="chi-button -primary -uppercase -px--4">Apply</button>
-              <button class="chi-button -primary -outline -bg--white -uppercase -px--4 -ml--2">Cancel</button>
-            </div>
           </div>
         </div>
-      </div>
-      <div class="chi-backdrop -closed">
-        <div class="chi-backdrop__wrapper">
-          <div class="chi-drawer -right -menu -position--absolute" id="drawer-2">
-            <div class="chi-drawer__header">
-              <span class="chi-drawer__title">Actions</span>
-              <button class="chi-button -icon -close" aria-label="Close">
-                <div class="chi-button__content">
-                  <i class="chi-icon icon-x"></i>
+
+        <!-- Drawer -->
+        <div class="chi-backdrop -closed">
+          <div class="chi-backdrop__wrapper">
+            <div class="chi-drawer -left -portal -menu -position--absolute" id="drawer-1">
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.